tutorials: Promote AvalancheGo installer as first/default method (#667)
ava-labs/avalanche-docs: docs/build/tutorials/nodes-and-staking/README.md

|  |  |
| :--- | :--- |

> | [**Run an Avalanche Node**](set-up-node-with-installer.md) | How to install and run AvalancheGo |

Reviewer: ```suggestion
| [**Run an Avalanche Node Using Install Script**](set-up-node-with-installer.md) | How to install and run AvalancheGo |
```

| [**Node Backup and Restore**](node-backup-and-restore.md) | Back up important files to be able to restore your node |
| [**Set up Node Monitoring**](setting-up-node-monitoring.md) | Set up infrastructure to monitor an instance of AvalancheGo |
| [**Add a Validator**](add-a-validator.md) | Add a node to the [Primary Network](../../../learn/platform-overview/README.md) |

> | [**Run a node manually, or from source**](run-avalanche-node.md) | How to install without the installer script |

Reviewer: ```suggestion
| [**Run an Avalanche Node Manually**](run-avalanche-node.md) | How to install without the installer script |
```
